cypress/elements/navigation.ts

```diff
@@ -0,0 +1,3 @@
+export const NAVIGATION = {
+  categoriesList: "[data-testid*='categoriesList']",
```

Comment:
Wouldn't be better to name it the categoryListButtons? And the same with other elements?